File: src/php/RandomMemoryBenchmark.php
<?php

namespace EntityFX\NetBenchmark\Core\Generic {

    use EntityFX\NetBenchmark\Core\Writer;
    
    class RandomMemoryBenchmark extends RandomMemoryBenchmarkBase {

        protected function BenchImplementation() {
            return $this->BenchRandomMemory();
        }

        protected function PopulateResult($benchResult, $result) {
            $benchResult["Points"] = $result["Average"] * $this->Ratio;
            $benchResult["Result"] = $result["Average"];
            $benchResult["Units"] = "MB/s";
            $benchResult["Output"] = $result["Output"];
            return $benchResult;
        }
    }
}

File: src/python/entityfx/random_memory_benchmark.py

from entityfx.random_memory_benchmark_base import RandomMemoryBenchmarkBase

class RandomMemoryBenchmark(RandomMemoryBenchmarkBase):
    
    def benchImplementation(self) :
        return self.benchRandomMemory()
    
    def populateResult(self, bench_result, result) :
        bench_result["Points"] = result["Average"] * self.ratio
        bench_result["Result"] = result["Average"]
        bench_result["Units"] = "MB/s"
        bench_result["Output"] = result["Output"]
        return bench_result